user/src/styles/pages/index.module.scss

```diff
@@ -0,0 +1,32 @@
+
+
+.structure {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: 200px;
+  justify-content: center;
+  left: 302px;
+  
+  
+}
+.content{
+  
+  
+  color:rgba(40, 141, 122, 1)
+  
+}
+.title{
+  color:rgba(79, 79, 79, 1);
+  font-weight: 2000;
+}
+.button{
+  color:rgba(31, 100, 133, 1);
+  border-radius: 8px;
+  font-weight: 2000;
+}
+
+.header-item{
+  width:508px;
```

Comment:
Use `max-width` instead of `width`. `width` breaks the responsive behavior of the general container of the page.